workflow-runtime-android/src/main/java/com/squareup/workflow1/android/AndroidRenderWorkflow.kt

```diff
@@ -291,16 +293,9 @@ public fun <PropsT, OutputT, RenderingT> renderWorkflowIn(
 ): StateFlow<RenderingT> {
   val restoredSnap = savedStateHandle?.get<PickledTreesnapshot>(KEY)?.snapshot
 
-  // Add in Compose's AndroidUiDispatcher.Main by default if none is specified.
```

Comment:
Would it be too magical to use reflection to look for `AndroidUiDispatcher.Main` and default to it if available?

Reply:
I think probably?